README.md

```diff
@@ -31,14 +40,22 @@ docker run -p 8090:8090 -d \
     -e MANTA_TIMEOUT=4000 \
     -e MANTA_METRIC_REPORTER_MODE=JMX \
     -e MANTA_HTTP_RETRIES=3 \
+    -e KEYSTORE_PATH=//Absolute path to the keystore \ (OPTIONAL. Required only for TLS)
+    -e KEYSTORE_PASS=XXXXXXXX \ (OPTIONAL. Required only for TLS)
+    -e JETTY_SERVER_PORT=8090 \
+    -e JETTY_SECURE_SERVER_PORT=8443 \(OPTIONAL. Required only if the above KEYSTORE_PATH and KEYSTORE_PASS are set)
```

Comment:
In the code you use the environment variable `JETTY_SERVER_SECURE_PORT`. Can you fix the code or the readme so that they use the same value?